src/main/java/robot/Operator.java

```diff
@@ -105,27 +112,32 @@ private void wheelControl() {
         if (getWristAngle() > (WRIST_STARTUP - OVERRIDE_TOLERANCE)) {
             System.out.println("Wheels cannot spin as the wrist is too close to the arm");
             wheels.stopWheels();
+
+            return; //Exits from the function before the speed can be set
```

Comment:
I saw this was taken out, but this is necessary so the set speed doesn't keep getting set back to the `WHEELS_SPEED_OUT` after the if statement completes.